.github/ISSUE_RESPONSES/comment.md

```diff
@@ -0,0 +1,5 @@
+## Thank you!
+
+We appreciate your contribution to this project! Please be sure to read the **contributing guide** to learn how you can help make this project better!
```

Comment:
Where is that contribution guide? I don't think I have one 😅

Reply:
Do you have a simple example?

Reply:
GitHub's contribution guide. I will get one of them.